content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -60,6 +60,10 @@ spec:
     name: pvc-1
 ```
 
+{{< note >}}
+The capacity value for `spec.resources.requests.storage` must be specified and must be >= the capacity of the source volume.
+{{< /note >}}
```

Comment:
I'm wondering what happens if you try to create a clone with a storage request that is too small. Does the API server reject the creation? Or does something else happen.

Reply:
nit: the style guide recommends writing in the [active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice).